tools/vscode-debugging/conf/launch.json

```diff
@@ -0,0 +1,49 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "interactive",
+            "type": "go",
+            "request": "launch",
+            "mode": "auto",
+            "preLaunchTask": "pre-debug",
+            "program": "${workspaceFolder}/cmd/launcher",
+            "args": [
+                "interactive",
+                "--osquery_flag=allow-unsafe",
```

Comment:
Do we still need this? I'd expect launcher to handle it

Reply:
honestly, I don't remember why I put that there, launcher interactive seems to work without it, removed

Reply:
I would assume it dates to when we invoked osquery interactive with tables.ext. But launcher interaction should abstract it.